pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -105,5 +107,18 @@ func status() registry.State {
 		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
 	}
 
+	// Ensure user is either a Windows Administrator or a Hyper-V Administrator.
+	adminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")`)
+	adminCheckOut, _ := adminCheckCmd.CombinedOutput()
+
+	hypervAdminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(([System.Security.Principal.SecurityIdentifier]::new("S-1-5-32-578")))`)
+	hypervAdminCheckOut, _ := hypervAdminCheckCmd.CombinedOutput()
+
+
+	if (strings.TrimSpace(string(adminCheckOut)) != "True") && (strings.TrimSpace(string(hypervAdminCheckOut)) != "True") {
+		exit.Error(reason.DrvNeedsAdministrator,"", errors.New("Hyper-v commands have to be run as an Administrator"))
```

Comment:
I belive this must return registry.State instead of exit.

doesn't this Exit minikube if it is on auto-select ? 

for exmaple if minikube is trying to evaluate docker and hyperv, to auto select one, if it comes to this code it wont go to Docker evaluation ....

Reply:
Oh yes, that would exit and not proceed ahead to the Docker driver. I have fixed this. Thanks!